project/Build.scala

```diff
@@ -819,6 +819,12 @@ object Build {
           normalizedName := "sbt-scalajs",
           bintrayProjectName := "sbt-scalajs-plugin", // "sbt-scalajs" was taken
           sbtPlugin := true,
+          sbtVersion in pluginCrossBuild := {
+            scalaVersion.value match {
+              case v if v.startsWith("2.10") || v.startsWith("2.11") => "0.13.17"
```

Comment:
The 2.11 case makes no sense, since this project can never be built for 2.11. I suggest simply removing `|| v.startsWith("2.11")`.

Also we usually include an additional dot on such tests in this codebase, so please use `v.startsWith("2.10.")` instead of `v.startsWith("2.10")`.

Reply:
Fixed